lib/FlagDependencyExportsPlugin.js

```diff
@@ -45,8 +45,7 @@ class FlagDependencyExportsPlugin {
 					if(exportDeps) {
 						exportDeps.forEach((dep) => {
 							const depIdent = dep.identifier();
-							let array = dependencies[depIdent];
-							if(!array) array = dependencies[depIdent] = [];
+							const array = dependencies[depIdent] = dependencies[depIdent] || [];
```

Comment:
You could change the return and second if to an else if like
```
const depIdent = dep.identifier();
const array = dependencies[depIdent];
if (!array) {
  dependencies[depIdent] = [module]
} else if (array.indexOf(module) < 0) {
  array.push(module);
}
```
but either way works.